openapi/src/openapi-src.json

```diff
@@ -600,7 +600,25 @@
                 "operationId": "getDataset",
                 "parameters": [
                     {
-                        "$ref": "#/components/parameters/questionnaireId"
+                        "name": "questionnaireId",
+                        "in": "path",
+                        "required": true,
+                        "description": "The id of the specific questionnaire instance. Format UUIDv4.",
```

Comment:
what's the reasoning behind moving `questionnaireId` out of the `components` object? Could we not have both the `questionnaireId` and the `Accept-Version` in the `components` object for consistency?

Reply:
Another clunky bit... I had to being this inline to allow it to have a different `example` id. Speccy complained when I had it with as a `$ref` with a sibling `example` keyword.

As for the `Accept-Version` being inlined too... I did have this as a param definition, however the semvers are specific to the endpoint, so there wasn't much reusability to be had.